services/workflows-service/prisma/schema.prisma

```diff
@@ -287,7 +287,8 @@ model Customer {
   country                     String?
   language                    String?
   websiteUrl                  String?
-
+  config                      Json?
```

Comment:
Lior i think we should have a separate `subscriptions` jsonb because it will be easier to enforce access - for example, we might have in config stuff that we don't want the customer to have access to - but subscription he can change however he see fit